kategory-test/src/main/kotlin/kategory/generators/Generators.kt

```diff
@@ -55,3 +55,8 @@ fun genIntPredicate(): Gen<(Int) -> Boolean> =
                     { it % absNum == absNum - 1 })
             )
         }
+
+fun <B> genOption(genB: Gen<B>): Gen<Option<B>> =
```

Comment:
This method could be implemented using genApplicative(). In any case, to properly test Option we'd need to generate at all possible cases. What I'd suggest is to randomise None to a low percent by capturing a Gen.int() and checking modulus 20 for 5% chance.